tests/Unit/Libraries/Tuf/HttpLoaderTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/**
+ * @package     Joomla.UnitTest
+ * @subpackage  Tuf
+ *
+ * @copyright   (C) 2019 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Tests\Unit\Libraries\Tuf;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Http\Http;
+use Joomla\CMS\Http\HttpFactoryInterface;
```

Comment:
```suggestion
use Joomla\CMS\Http\Http;
```
Remove unused imports to make php-cs-fixer happy.